src/containers/App.js

```diff
@@ -1,15 +1,16 @@
 import React, { Component } from "react";
-import { Router } from "react-router-dom";
-import history from "../routes/History";
-import Routes from "../routes/Routes";
+//import { Router } from "react-router-dom";
+//import history from "../routes/History";
+//import Routes from "../routes/Routes";
 import "./App.scss";
+import Home from "./Home/Home";
 
 class App extends Component {
   render() {
     return (
-      <div>
-        <Router history={history}>{Routes}</Router>
```

Comment:
Please redo this changes, it allow us to navigate back and forward in the app routes.

Reply:
Yep, mate! I just forget routes and history folder, I though that we gonna implement it manually! nice catch btw :+1: